src/main/java/duke/command/AllCommand.java

```diff
@@ -0,0 +1,58 @@
+package duke.command;
+
+import java.util.Arrays;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import javafx.util.Pair;
+
+/**
+ * All Command for Duke application.
+ *
+ * @author Farrel Dwireswara Salim
+ */
+public class AllCommand implements Command {
+    private final String[] tags;
```

Comment:
Might be useful to create tags as its own class to prevent [Primitive Obsession](https://refactoring.guru/smells/primitive-obsession)!